src/adaptations/device-layer/nRF5/BLEManagerImpl.cpp

```diff
@@ -278,6 +278,21 @@ void BLEManagerImpl::_OnPlatformEvent(const WeaveDeviceEvent * event)
         WeaveLogError(DeviceLayer, "BLEManagerImpl: Out of buffers during WoBLE RX");
         break;
 
+    // If WOBLE_DISABLE_ADVERTISING_WHEN_PROVISIONED is enabled, and there is a change to the
```

Comment:
This looks the same, if not nearly the same as for ESP32. Is there an argument for having a common, base implementation?

Reply:
Eventually, yes.  It requires some thinking about how to best factor out the common logic, which I haven't had time to do yet.